src/core/Wizard/Step.tsx

```diff
@@ -80,7 +80,12 @@ export const Step = (props: StepProps) => {
       )}
 
       {type !== 'workflow' && (
-        <span className='iui-wizards-step-title'>{title}</span>
+        <span
+          className='iui-wizards-step-title'
+          style={{ width: `${100 / totalSteps}vw` }}
```

Comment:
I think it can cause some bugs because most of the cases wizard won't take the whole screen e.g. page having an expanded sidenav and maybe some container with padding then 100% wizard width != 100vw

Reply:
Yeah, that was the reason I suggested to have % in css (which broke things a little...).

Reply:
Then why didn't we try to fix those broken things? 😄

% is obviously better, but maybe there's a better solution?

Reply:
CSS fixed in https://github.com/iTwin/iTwinUI/pull/119. Using percentage in Step now